test/common.sh

```diff
@@ -66,9 +66,22 @@ function knative_setup() {
     install_istio
   fi
 
-  local serving_version=${KNATIVE_SERVING_VERSION:-latest}
-  header "Installing Knative Serving (${serving_version})"
+  # Serving and Eventing 'latests' is based on branch context
+  # On `main` it means 'nightly' manifests
+  # On `release-*` it means corresponding `release-*` manifests
+  # The described behavior is achieved through hack/library.sh start_latest_knative_serving()
+  local serving_version="latest"
+  local eventing_version="latest"
+
+  # PR check variant triggering presubmit-integration-tests-latest-release
+  # should check current client main branch code on latest avaible release
+  # e.g. PR to 'main' on Serving and Eventing 1.3.0
+  if [[ -n ${LATEST_RELEASE} ]]; then
```

Comment:
```suggestion
  if [ -n "${LATEST_RELEASE}" ]; then
```

Just a proposal to be POSIX compliant, but since we already went done the Bash rabbit hole, don't mind to ignore this proposal.

Reply:
But it would be consistent to the usage in the rest of the script

Reply:
Or something like to also check the value of the env var:

```
if [ "xtrue" == x"${LATEST_RELEASE}" ]; then
```